tests/specs/clear.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+var test = require('tape')
+
+var dbFactory = require('../utils/db')
+
+test('has "clear" method', function (test) {
+  test.plan(1)
+
+  var db = dbFactory()
+  var store = db.hoodieApi()
+
+  test.is(typeof store.clear, 'function', 'has method')
+})
+
+test('removes existing db', function (t) {
+  t.plan(3)
+
+  var db = dbFactory()
+  var store = db.hoodieApi()
+
+  var localObj1 = {
+    _id: 'cleanTest',
+    cnt: 'cleanTest dummy doc'
+  }
+  store.add(localObj1)
+  store.on('clear', function () {
+    t.pass("store 'clear' event was triggered")
+  })
+
+  db.get('cleanTest')
+    .then(function (results) {
+      t.equal(results._id, 'cleanTest', 'test data is in place')
+
+      store.clear().then(function () {
+        t.pass('test passed, db was destroyed')
+      })
+    }
+  )
+
+  .catch(function (err) {
+    t.ok(err instanceof Error, 'rejects error')
```

Comment:
This is a fourth test assertion, you only plan 3 above.

This code is likely not being run ever, so we might want to remove it.

Since this is the `catch()` for `db.get()`, which we are not testing here, we don’t have to worry about testing an error case.

If this code needs a `catch()` to run issue free, I’d just remove the `t.ok` assertion.

(note that this is the first time I’m reviewing code in this part of the Hoodie world, so maybe get a second opinion from @zoepage or @gr2m :)